src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Ssl.cs

```diff
@@ -305,7 +305,7 @@ internal static unsafe void SslCtxSetAlpnProtos(SafeSslHandle ctx, List<SslAppli
             {
                 if (protocols.Count == 1 && protocols[0] == SslApplicationProtocol.Http2)
                 {
-                    cfProtocolsRefs = s_cfAlpnHttp211Protocols;
+                    cfProtocolsRefs = s_cfAlpnHttp2Protocols;
```

Comment:
@wfurt, can you confirm that this was a mistake and should be fixed as I've done?

Reply:
yes. this looks like correct change @stephentoub 
Compiler warnings are useful.

Reply:
Thanks, @wfurt.  How important is this?  Is it ok not being in 3.0?

Reply:
The old string would be more liberal and also advertise 11 as fallback when it should not. 
So I think the risk is small unless @bartonjs think otherwise.